### PR TITLE
Milestone 01: To-Do list: list structure

### PR DESCRIPTION
**Pull request by mavericksbalitaan**
## To-Do list: list structure

This branch includes:
- [ ] Set up a new project with webpack that is based on the webpack exercise you have already completed.
- [ ] Create an index.html file and write your HTML markup here. Create an empty To-Do List placeholder div or ul element. The index.html file must be set as a template using the HTML Webpack Plugin.
- [ ] Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].
- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ]  On page load renders the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- [ ] Create a style.css and set rules for the To-Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
- [ ] All your source files (index.html, index.js, and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    const p = document.createElement('p');
+    p.textContent = el.description;
+    button[el.index] = document.createElement('button');
+    button[el.index].setAttribute('id', el.index);
+    button[el.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+    li[el.index].append(input[el.index], p, button[el.index]);
+    listContainer.append(li[el.index]);
+  });
+};
```
Comment (Mihndim2020) [inline src/index.js:111]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for this project, " `On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.` ", please make sure  the tasks appear in order of the index values for each task, that is from the smallest to the biggest(`ascending order`).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @mavericks-db, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Descriptive `PR` summary,
:heavy_check_mark: Clean and organized code. 

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -90,8 +90,9 @@ const listArray = [
 // Dynamically Created List of Tasks Function
 const loopTask = () => {
   const listContainer = document.querySelector('.listContainer');
-  const sorted = listArray.sort((a, b) => (a.index < b.index ? 1 : -1));
-  sorted.forEach((el) => {
+  // const sorted = listArray.sort((a, b) => (a.index < b.index ? 1 : -1));
+  // as per previous code reviewer to change the order
```
Comment (julie-ify) [inline src/index.js]: - [ ] **[OPTIONAL]** Kindly remove these commented codes because I clearly understand what happened here okay 😄

**Review by julie-ify — APPROVED** [review]
## APPROVED ✔️ ✔️

Hi @mavericks-db, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**